templates/server-deployment.yaml

```diff
@@ -70,11 +70,11 @@ spec:
                 fieldRef:
                   fieldPath: status.podIP
             - name: ENABLE_ES
-              value: "{{ $.Values.elasticsearch.enabled }}"
+              value: "{{ or $.Values.elasticsearch.enabled $.Values.elasticsearch.hosted }}"
             - name: ES_SEEDS
-              value: elasticsearch-master-headless
+              value: {{ $.Values.elasticsearch.host }}
```

Comment:
I think this breaks the default setup, since a host and port isn't provided in the base values.yaml :-)

Reply:
thank you, fixed!